website/docs/r/policy_gateway_policy.html.markdown

```diff
@@ -93,7 +93,7 @@ The following arguments are supported:
 * `rule` (Optional) A repeatable block to specify rules for the Gateway Policy. Each rule includes the following fields:
   * `display_name` - (Required) Display name of the resource.
   * `description` - (Optional) Description of the resource.
-  * `destination_groups` - (Optional) A list of destination group paths to use for the policy.
+  * `destination_groups` - (Optional) Set of group paths, IPs, IP ranges, or CIDRs that serve as the destination for this rule.
```

Comment:
It should be helpful specifying here that IP/ranges/cidrs are only supported starting NSX 3.0.

Reply:
Hi @annakhm, I have updated all of the documentation as requested.